flumine/backtest/simulated.py

```diff
@@ -185,10 +197,25 @@ def _process_price_matched(
                 else:
                     _size_matched = avail["size"]
                 _matched = [publish_time, avail["price"], round(_size_matched, 2)]
-                self._update_matched(_matched)
+                _matches.append(_matched)
             else:
                 break
 
+        if self.order.order_type.time_in_force == "FILL_OR_KILL":
+            if size_remaining == 0:
+                self._update_multi_matched(lst_data=_matches)
+            elif self.order.order_type.min_fill_size:
+                filled_size = sum(x[2] for x in _matches)
```

Comment:
To confirm my understanding this is also equal to `size-size_remaining`?

Reply:
Yes. I have updated the code to use your clearer/more efficient suggestion.